tutorials/rendering_plugins.md

```diff
@@ -45,7 +45,7 @@ To interact with the server-side scene, you'll need to write an
 See [Create System Plugins](createsystemplugins.html).
 
 To interact with the client-side scene, you'll need to write an
-[gz::gui::Plugin](https://gazebosim.org/api/gui/4.1/classignition_1_1gui_1_1Plugin.html),
+[gz::gui::Plugin](https://gazebosim.org/api/gui/8/classgz_1_1gui_1_1Plugin.html),
```

Comment:
If you add
```
     "${GZ-GUI_DOXYGEN_TAGFILE} = ${GZ-GUI_API_URL}"
```
in https://github.com/gazebosim/gz-sim/blob/142c447dc11c0f267388c52dd4ef79413ad6f047/CMakeLists.txt#L275, we should be able to use `\ref gz::gui::Plugin` here.

Reply:
Cool! That worked